### ISSUE TITLE
On the solution of section 3B - 27

### ISSUE THREAD

**Issue by lumliolum** [nehc0/Linear-Algebra-Done-Right-4th-Solutions#2]
In the solution of yours, you have assumed that $V$ is finite dimensional. But it's not given in the question. I got an alternative proof of the same. Let me know what do you think.

![image](https://github.com/user-attachments/assets/e7a52992-c1d4-43ba-bb44-328f01f4bd6b)

**Comment by nehc0** [comment]
Yes, you are right. The proof of V=N(T)+R(T) should not use dimension. Your proof of that part is nice.
